Tests/Common/Data/Custom/TradingEconomicsTests.cs

```diff
@@ -385,5 +385,46 @@ public void DecimalIsParsedCorrectly(string value, double? expected, bool inPerc
             // Cast inside since we can't pass in decimal values through TestCase attributes
             Assert.AreEqual((decimal?)expected, TradingEconomicsCalendar.ParseDecimal(value, inPercentage));
         }
+
+        [Test]
+        public void SerializeRoundTrip()
+        {
+            var settings = new JsonSerializerSettings { TypeNameHandling = TypeNameHandling.All };
+
+            var time = new DateTime(2020, 3, 20, 14, 0, 0);
+            var symbol = Symbol.CreateBase(
```

Comment:
The call to `Symbol.CreateBase` isn't necessary, since we don't have underlying `Symbol`s for this kind of custom data.

Do this instead: 
```
Symbol.Create(
    TradingEconomics.Calendar.UnitedStates.ExistingHomeSales,
    SecurityType.Base,
    Market.USA,
    baseDataType: typeof(TradingEconomicsCalendar)
);
```